src/computability/language.lean

```diff
@@ -17,6 +17,8 @@ over the languages.
 
 universes u v
 
+open set
```

Comment:
I'm not convinced this really makes things clearer

Reply:
Doesn't it, though? This file is about `list` and `set`, so let's open `list` and `set`.

Reply:
I'd argue it's a thin wrapper around list and set, which means it can be easy to get confused between the `language` API and the wrapped API that is only supposed to be used internally.

Reply:
What does that even mean? There is literally no lemma names in common between `language` and `set`/`list`.